### DISCUSSION TITLE
Cannot connect to host discord.com:443 problems

### DISCUSSION THREAD

**Discussion by lay4940** [Rapptz/discord.py#6362] [Q&A]
today I start to make a discord bot and I use this code

import discord
token = "blablablablablablsomethingtoken"
client = discord.Client()

@client.event
async def on_ready():
    print("일러났습니다")
    print(client.user)
    print('============================')

client.run(token)

and this came out Cannot connect to host discord.com:443
wot is the problems?

**Comment by bandwiches** [comment]
This is unrelated to the repo, this has to do with the host not being able to connect to discord.com.

**Comment by mesub7** [comment]
It means that your DNS doesn't have discord for some reason.

Try a different DNS

**Comment by Rapptz** [comment]
A DNS resolution error happens if DNS lookup fails. To fix this you need to set up a secondary DNS, instructions to do so depend on your operating system so you should search the internet for more information.
